spec/system/admin/products_v3/products_spec.rb

```diff
@@ -26,20 +26,31 @@
   describe "sorting" do
     let!(:product_b) { create(:simple_product, name: "Bananas") }
     let!(:product_a) { create(:simple_product, name: "Apples") }
+    let(:products_table) { "table.products" }
 
     before do
       visit admin_products_url
     end
 
-    it "Should sort products alphabetically by default" do
-      within "table.products" do
+    it "Should sort products alphabetically by default in ascending order" do
+      within products_table do
         # Gather input values, because page.content doesn't include them.
         input_content = page.find_all('input[type=text]').map(&:value).join
```

Comment:
Maybe this can be moved to a method now

Reply:
It's updated here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12449/commits/8e1218ae53509bbe7e9b6bef849dcb03a8341b65

Reply:
🤩